incubator/patroni/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: patroni
 description: 'Highly available elephant herd: HA PostgreSQL cluster.'
-version: 0.11.0
+version: 0.11.1
```

Comment:
Because this changes the `matchLabels` selector you can't upgrade from `0.11.0` to this version. Can you please increment the minor version.

Reply:
Done